src/js/account-page.js

```diff
@@ -1,32 +1,59 @@
-jQuery(document).ready(function($) {
+jQuery(document).ready(($) => {
+  /**
+   * Hide custom NL fields by default when country not NL.
+   *
+   * @param {String} addressType
+   */
+  function localizeAddressFields(addressType) {
+    const country = $(`#${addressType}_country`).val();
```

Comment:
Ik vraag mij een beetje af wat het beste werkt.
Je checkt nu op address_type en in sommige gevallen haal je de _country er eerst af om daarna weer op te zetten.

Je zou ook bij het benoemen van de addresstypes de _country toe kunnen voegen, dus `localizeAddressFields('shipping_country');` om daarna in deze functie puur te checken op `selector`. Zo kan je het weer hergebruiken als het om bijvoorbeeld een `_address` zou gaan.